### PR TITLE
delete case/by_xform_id couch view

### PR DESCRIPTION
**Pull request by czue**
should be merged/reindexed along with https://github.com/dimagi/commcare-hq/pull/6773

### CODE REVIEW

**Comment by czue** [comment]
@NoahCarnahan

File: corehq/apps/cleanup/management/commands/check_case_integrity.py [hunk 1]
```diff
@@ -1,106 +0,0 @@
-from collections import defaultdict
```
Comment (czue) [inline corehq/apps/cleanup/management/commands/check_case_integrity.py]: I don't think this script is useful anymore (and it's a bit of a mess). easier to just remove it then try and figure out how to port it over.

File: corehq/apps/cleanup/management/commands/generate_form_case_consistency_list.py [hunk 2]
```diff
@@ -1,161 +0,0 @@
-from datetime import datetime, timedelta
```
Comment (czue) [inline corehq/apps/cleanup/management/commands/generate_form_case_consistency_list.py]: same with this one. now that we have atomic submissions i think this is deprecated. also i think https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py sort of serves its original purpose. cc @dannyroberts @snopoke

**Comment by snopoke** [comment]
looks good

**Comment by dannyroberts** [comment]
very nice!

**Pull request merged**